src/main/resources/db/migration/V1_14__Alter_List_Of_Values.sql

```diff
@@ -0,0 +1,10 @@
+
+ALTER TABLE List_Of_Values ADD COLUMN external_reference VARCHAR(200);
+ALTER TABLE List_Of_Values ADD COLUMN external_reference_type VARCHAR(200);
+
+
+ALTER TABLE list_of_values
```

Comment:
minor: readability - please decide on either lower case or capitalised case across migration file

Reply:
changed to upper